content/swiftui/concepts/views/terms/button/button.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'Button'
+Description: 'A very basic component in the UI, which is used to interact and perform certain actions'
+Subjects:
+  - 'Sotfware Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Buttons'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **button** is a very basic component in the UI, which is used to interact and perform certain actions, such as logging into a page, or deleting a file etc.
+
+## Syntax
+
+```swift
+Button("Text inside the button"){
```

Comment:
```suggestion
Button("Text inside the button") {
```

Reply:
Does it have to be text? Can it be an image

Reply:
Yes, it can. I'll rephrase this one.

Reply:
Rephrased it.